tests/client/amo/components/TestAddonCompatibilityError.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import {
+  renderIntoDocument,
+  findRenderedComponentWithType,
+} from 'react-addons-test-utils';
+import { findDOMNode } from 'react-dom';
+import { Provider } from 'react-redux';
+
+import AddonCompatibilityError from 'amo/components/AddonCompatibilityError';
+import createStore from 'amo/store';
+import {
+  INCOMPATIBLE_FIREFOX_FOR_IOS,
+  INCOMPATIBLE_NOT_FIREFOX,
+  INCOMPATIBLE_OVER_MAX_VERSION,
+  INCOMPATIBLE_UNDER_MIN_VERSION,
+} from 'core/constants';
+import { signedInApiState } from 'tests/client/amo/helpers';
+import { getFakeI18nInst } from 'tests/client/helpers';
+import I18nProvider from 'core/i18n/Provider';
+
+
+describe('AddonCompatibilityError', () => {
+  function render({ ...props }) {
+    const api = {
+      ...signedInApiState,
+      lang: props.lang || signedInApiState.lang,
```

Comment:
Well, I should clarify that it only fall back if `props.lang` is undefined. If `props.lang` is `null` or an empty string, it won't fall back.